src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1201,7 +1201,7 @@ public static bool IsParameterTypeContext(this SyntaxTree syntaxTree, int positi
 
             if (token.IsKind(SyntaxKind.OutKeyword) &&
                 token.Parent.IsKind(SyntaxKind.Argument) &&
-                ((ArgumentSyntax)token.Parent).Declaration != null)
```

Comment:
I have a different perspective, I check for things I am looking for without making assumptions. Things might change and ```out``` keyword might become a valid part of the argument syntax in a different position. I am interested in the one that is used as RefOrOutKeyword. Checking for this explicitly makes code self-explanatory and robust.